Gemfile

```diff
@@ -10,7 +10,13 @@ gem 'gds-sso', '~> 11.2'
 gem 'gds-api-adapters', '~> 40.1.0'
 gem 'govspeak', '~> 3.4.0'
 gem 'govuk_admin_template', '4.2.0'
-gem "govuk_content_models", "~> 42.0.1"
+
```

Comment:
I think we've kind of ping-pong-ed on this before with adding/removing the dev env stuff. I think we agreed to not include it in the Gemfile and just do it locally if we have to for local development purposes...